src/main/java/com/dap/DailyArtPrompt/service/ImageService.java

```diff
@@ -2,14 +2,26 @@
 
 import com.dap.DailyArtPrompt.model.Image;
 import com.dap.DailyArtPrompt.model.ImageResponse;
+import org.springframework.stereotype.Component;
 import org.springframework.web.client.RestTemplate;
 
+@Component
 public class ImageService {
-    public Image getImage() {
-        final String uri = "https://dog.ceo/api/breeds/image/random";
-        RestTemplate restTemplate = new RestTemplate();
-        ImageResponse image = restTemplate.getForObject(uri, ImageResponse.class);
 
-        return new Image(image.getMessage());
+    private final RestTemplate restTemplate;
+
+    public ImageService(RestTemplate restTemplate) {
```

Comment:
have you actually tested your service? AKA: used insomnia or your FE to call it?